src/main/java/BasketInformations.java

```diff
@@ -0,0 +1,98 @@
+package com.company;
+
+import java.util.HashMap;
+
+/**
+ * This class is a basket
+ */
+public class BasketInformations {
```

Comment:
This comment does not add anything new about the class, perhaps you could document a little bit about the behaviour and what it does.
**Example:** 

"This class store local producs that are in the basket and offer methods to access the total value of the products."